themes/default_symbols.fish

```diff
@@ -33,6 +33,9 @@ set FLSYM_ROOT_USER         "\u2192"
 # Symbol for VFISH segment
 set FLSYM_VFISH             "\u2635"
 
+# Symbol for CONDA segment
+set FLSYM_CONDA             "\U0001F40D"
```

Comment:
Sure; I made it `conda:` in the tty-compatible theme, but I'm open to changing it in general.  (FWIW I use linux and it renders fine with sauce code pro as a font.)

Trigrams are a nice option, but I also worry that they're too similar to the symbol used by vfish.

How about ⚕ (`\u2695`)?